tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Send Email.3540.feature

```diff
@@ -0,0 +1,52 @@
+﻿Feature: Send Email Confirmation Link
+    As a Authority User
+    I want to send confirmation link to users after creating their accounts
+    So Users can complete their registration
+
+Background:
+	Given Organisations exist
+		| Name           | OdsCode |
+		| Organisation 1 | Ods 1   |
+	And Users exist
+		| Id     | OrganisationName | FirstName | LastName | Email                | PhoneNumber | Disabled | Password        | OrganisationFunction |
+		| 012345 | Organisation 1   | Penny     | Lane     | PennyLane@email.com  | 01234567890 | false    | S0mePa$$w0rd    | Buyer                |
+		| 123456 | Organisation 1   | Post      | Pat      | PostmanPat@email.com | 12345678901 | false    | An0therPa$$w0rd | Authority            |
+
+@3540
+Scenario: 1. User receives email after their account is created
+	Given an user is logged in
+		| Username             | Password        | Scope        |
+		| PostmanPat@email.com | An0therPa$$w0rd | Organisation |
+	When a POST request is made to create a user for organisation Organisation 1
+		| FirstName | LastName   | PhoneNumber | EmailAddress             | OrganisationName |
+		| Bob       | Bobkovitch | 0123456789  | bob.bobkovitch@email.com | Organisation 1   |
+	Then a response with status code 200 is returned
+	And an email containing the following information is sent
```

Comment:
This part of the test seems superfluous, as the test is generating and sending the email, then checking the result, without interacting with the api.

Reply:
I disagree - The previous steps are necessary for the integration tests as creating the user is a trigger for the email to be sent. 

This also provides more context. Now you know that emails are only being sent whenever a user is successfully created as opposed to just: 
- send email with some data
- confirm email received, containing correct data.

Reply:
As per conversation, Chris is implementing this, so wait for his changes, and then remove thee email is sent step.